apache-rat-tools/src/test/java/org/apache/rat/tools/NamingTest.java

```diff
@@ -181,10 +181,4 @@ private void assertContains(String expected, List<String> actual) {
     private void assertNotContains(String expected, List<String> actual) {
         assertFalse(actual.contains(expected), () -> "Contains "+expected);
     }
-
-    @Test
```

Comment:
If I'm not too mistaken I added these tests to validate the NPE-safeness of the Naming class? Is that not needed anymore?